_posts/2023-06-23-podman-for-windows.adoc

```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title: Podman for Windows
+date: 2023-06-23
```

Comment:
Bump the date to today before we merge this (also, please rename the file to match the date)

Reply:
Ack.